docs/sources/flow/tutorials/collecting-prometheus-metrics.md

```diff
@@ -10,16 +10,33 @@ Grafana Agent is a telemetry collector with the primary goal of moving telemetry
 # Prerequisites
 
 * [Docker](https://www.docker.com/products/docker-desktop)
-* Clone the [Agent Repository](https://github.com/grafana/agent) `git clone git@github.com:grafana/agent.git`
 
 # Scraping the Agent
 
-To quickly spin up an example environment, run the following: `CONFIG_FILE=agent.flow docker-compose -f ./assets/docker-compose.yaml up` with the [flow file](../assets/flow_configs/agent.flow). Allow the service to run for a few minutes, then navigate to the [metrics browser](http://localhost:3000/explore?orgId=1&left=%5B%22now-1h%22,%22now%22,%22Mimir%22,%7B%22refId%22:%22A%22,%22instant%22:true,%22range%22:true,%22exemplar%22:true,%22expr%22:%22agent_build_info%7B%7D%22%7D%5D).
+Run the following `curl https://raw.githubusercontent.com/grafana/agent/main/docs/sources/flow/tutorials/assets/runt.sh -O && bash ./runt.sh agent.flow`. 
+
+The `runt.sh` script does:
+
+1. Downloads the configs necessary for Mimir, Grafana and the Grafana Agent. 
+2. Downloads the docker image for Grafana Agent explicitly.
+3. Runs the docker-compose up command to bring all the services up.
+
+Allow the Grafana Agent to run for two minutes, then navigate to [Grafana](http://localhost:3000/explore?orgId=1&left=%5B%22now-1h%22,%22now%22,%22Mimir%22,%7B%22refId%22:%22A%22,%22instant%22:true,%22range%22:true,%22exemplar%22:true,%22expr%22:%22agent_build_info%7B%7D%22%7D%5D). 
 
 ![](../assets/agent_build_info.png)
 
 This example scrapes the Grafana Agent's `http://localhost:12345/metrics` endpoint and pushes those metrics to the Mimir instance.
 
+Navigate to `http://localhost:12345/graph` to view the Grafana Agent Flow UI. 
+
+![](../assets/graph.png)
+
```

Comment:
```suggestion
The Agent displays the component pipeline in a dependency graph.  See the [Scraping component](url) and [Remote Write ](url) sections for details about the components. 
```

Reply:
Sorry, not sure yet about the hugo syntax...  I just want them to know that the details exist in the topic.

Reply:
We should probably think about how best to arrange the topics in these tutorials -- a longer term goal. I wonder if we might want to create a pattern where we use expanding text near the top of a tutorial that would be titled "Understand the configuration" -- and we'd include the explanations of the components there.  We'd probably need to work with Jack, though?

Reply:
If we have expanding tags would love to use them, or spoiler style. I was going to put the answers to the tutorials in spoiler style sections but couldn't do it without custom CSS.